src/proc/syncer-processing-block.cpp

```diff
@@ -10,63 +10,98 @@
 
 namespace librealsense
 {
-    syncer_process_unit::syncer_process_unit( std::initializer_list< bool_option::ptr > enable_opts )
-        : processing_block("syncer"), _matcher((new timestamp_composite_matcher({})))
-        , _enable_opts( enable_opts.begin(), enable_opts.end() )
-    {
-        _matcher->set_callback([this](frame_holder f, syncronization_environment env)
+syncer_process_unit::syncer_process_unit( std::initializer_list< bool_option::ptr > enable_opts,
+                                          bool log )
```

Comment:
I don't see `_log` being initialized... is it even in use?

Reply:
done

Reply:
I still don't see initialization of `_log`...

Reply:
I put it on struct declaration : bool log = true;

Reply:
Avishag, class syncer_process_unit has a member called `_log` that's not initialized...

Reply:
removed _log